src/ontology/components/mappings.owl

```diff
@@ -28,6 +29,24 @@
     
 
 
+    <!-- https://w3id.org/sssom/author_id -->
+
+    <owl:AnnotationProperty rdf:about="https://w3id.org/sssom/author_id"/>
```

Comment:
why is this its own property? shouldn't this be dc:creator or dc:contributor?

Reply:
Because that’s what the [SSSOM spec mandates](https://mapping-commons.github.io/sssom/spec/#predicate-is-an-annotation-property):

> All mapping level metadata (sssomMetadata) gets converted into OWLAnnotation objects which are materialised as axiom annotations on the mapping annotation assertion [...]
>
> ```
> AnnotationAssertion(sssomMetadata P, S, O)
> ```
>
> Where sssomMetadata is a sequence of OWL Annotations objects like:
>
> ```
> Annotation(Q1,V1) Annotation(Q2,V2) ... Annotation(Qn,Vn)
> ```
>
> where Qi is a SSSOM metadata element and Vi is an annotation value.
> [...]
> Example:
>
> ```
> AnnotationAssertion(Annotation(sssom:creator_id <https://orcid.org/0000-0002-7356-1779>) Annotation(sssom:mapping_justification semapv:LexicalMatching) skos:exactMatch <http://purl.obolibrary.org/obo/HP_0009894> <http://purl.obolibrary.org/obo/MP_0000019>)
> ```

Reply:
oh boy...

Reply:
Some of this have mappings to standard properties, like creator_id: https://github.com/mapping-commons/sssom/blob/master/src/sssom_schema/schema/sssom_schema.yaml#L317

Feel free to request more such mappings to standard properties. But "author" is, for example, more fine-grained than dcterms:contributor so unlikely to be a suitable match.